content/learn/book/plugin-development/_index.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Plugin Development"
```

Comment:
```suggestion
title = "Building Bevy's Ecosystem"
```

Reply:
Not 100% sold on the title here, but I think we need to indicate that this isn't directly intended for use in your own games.